.github/workflows/ci.yml

```diff
@@ -508,10 +510,27 @@ jobs:
         exclude_file: .codespell.exclude
 
 
-
   style-check:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
     - uses: editorconfig-checker/action-editorconfig-checker@main
     - run: editorconfig-checker
+
+
+  check-coding-style:
```

Comment:
This should only be run if the source files have changed. You can use the `game_code` filter for this.

Reply:
I _think_ I fixed it. You should probably have a second look though.